manual/src/cmds/unified-options.etex

```diff
@@ -684,9 +684,9 @@ Linux AMD 64, they must contain only position-independent code).
 \item["-short-paths"]
 When a type is visible under several module-paths, use the shortest
 one when printing the type's name in inferred interfaces and error and
-warning messages. Identifier names starting with an underscore "_" or
-containing double underscores "__" incur a penalty of $+10$ when computing
-their length.
+warning messages. Identifier names starting with an underscore "_", a
+dollar "$" or containing double underscores "__" incur a penalty of
```

Comment:
Shouldn't this be escaped?

Reply:
`tools/texquote2` does escape it automatically.